src/main/java/bio/terra/janitor/service/cleanup/flight/GoogleBlobCleanupFlight.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.janitor.service.cleanup.flight;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.janitor.db.JanitorDao;
+import bio.terra.stairway.Flight;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.RetryRuleFixedInterval;
+import org.springframework.context.ApplicationContext;
+
+/** A Flight to cleanup Blob in GCP. */
+public class GoogleBlobCleanupFlight extends Flight {
```

Comment:
@wchamber I can also make an abstract ResourceCleanupFlight and all resource externs that class. What do you prefer?

Reply:
Hmmm. I'm not sure we get a lot from having a base flight class. I guess it would handle the initial/cleanup steps, some default retry, but that's not a lot. WDYT?
I'm ok either way, and we can always change our mind later.